src/Init/WFTactics.lean

```diff
@@ -22,7 +22,7 @@ macro_rules | `(tactic| decreasing_trivial) => `(tactic| linarith)
 -/
 syntax "decreasing_trivial" : tactic
 
-macro_rules | `(tactic| decreasing_trivial) => `(tactic| simp (config := { arith := true }); done)
+macro_rules | `(tactic| decreasing_trivial) => `(tactic| (try simp (config := { arith := true })); done)
```

Comment:
Should this instead be
```suggestion
macro_rules | `(tactic| decreasing_trivial) => `(tactic| (simp (config := { arith := true, failIfUnchanged := false })); done)
```
Presumably you don't want to catch any and all failures